project.go

```diff
@@ -81,6 +81,19 @@ type (
 		APIResponse
 		Projects []*Project `json:"projects"`
 	}
+
```

Comment:
Please add annotations to make `gofmt` happy

Reply:
Also, please add Response -> `ProjectEventLogEntriesResponse` to the structure name, to make it constant with the rest of the code.

Reply:
Fixed in comit 848b4bf